code/__DEFINES/subsystems.dm

```diff
@@ -46,8 +46,8 @@
 #define INIT_ORDER_DBCORE 18
 #define INIT_ORDER_BLACKBOX 17
 #define INIT_ORDER_SERVER_MAINT 16
-#define INIT_ORDER_JOBS 15
-#define INIT_ORDER_EVENTS 14
+#define INIT_ORDER_EVENTS 15
```

Comment:
I had to make events initialize before jobs so jobs could check the holiday when picking roundstart species